### ISSUE TITLE
Background CSS images need alt text

### ISSUE THREAD

**Issue by mgifford** [cds-snc/digital-canada-ca#59]
Just because you're using CSS to display an image doesn't mean that you can skip the alt text. The images in the blog (both on the list of pages and the hero images on individual pages) in most cases should have a description. 

https://digital.canada.ca/blog/

There are ways to do this, but the images convey meaning and context even if the automated tools don't highlight it as an issue.

**Comment by dsamojlenko** [comment]
Changes underway in #67 

Thanks for this feedback, Mike.

**Comment by sboots** [comment]
Referenced in, <https://digital.canada.ca/2018/01/16/challenging-our-assumption/>

Thanks again for the feedback @mgifford!
